pkg/clusteragent/custommetrics/store.go

```diff
@@ -41,6 +42,11 @@ type configMapStore struct {
 	cm        *v1.ConfigMap
 }
 
+// GetHPAConfigmapName returns the name of the ConfigMap used to store the state of the Custom Metrics Provider
+func GetHPAConfigmapName() string {
+	return config.Datadog.GetString("hpa_configmap_name")
+}
```

Comment:
This function could just be called `GetStorageConfigMapName` or `GetConfigMapName`. It's already in the `custommetrics` package so it's pretty clear already that i's storing metrics.